### PR TITLE
Design Teardown Project

### PR DESCRIPTION
**Pull request by DcRonan**
### A greyscale heat map of Smashing Magazine.

1. Here we made the text disappear to create a nicer look.
2. We used grid, flex, and float.
3. Used line-height to space out the text.
4. Used transform effects.
5. Created variables for background-colors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,791 @@
...
+                    </p>
+                </div>
+
+                <div class="arrow-button">
+                    <span><i class="fas fa-arrow-right"></i> </span>
+                </div>
+            </div>
+
+            <div class="div2 class1">
+
```
Comment (bertil291utn) [inline index.html]: ## Article tag

It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

- Change all `<div class="div class1">` tags to `<article>` tag in HTML file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,791 @@
...
+                            architecto, corrupti at esse. </span>
+                    </article>
+                    <div class="arrow-button">
+                        <span><i class="fas fa-arrow-right"></i> </span>
+                    </div>
+
+                </div>
+                <div class="flex-box-2 classhover">
+                    <h2>
+                        <span>
```
Comment (bertil291utn) [inline index.html]: ## Article tag

It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

- Change all `<div class="flex-box- classhover">` tags to `<article>` tag in HTML file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,791 @@
...
+                        </span>
+                    </button>
+                    <p>
+                        <span>
+                            Lorem lorem lorem lorem
+                        </span>
+
+                    </p>
+                </div>
+            </div>
```
Comment (bertil291utn) [inline index.html]: ## Article tag

It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

- Change all elements inside latest post and latest post-2 classes tags to `<article>` tag instead just divs

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+}
+
+.flexcontainer-1 {
+  display: flex;
+}
+
+.class1 {
+  padding: 90px 20px;
+  color: white;
+  display: flex;
```
Comment (bertil291utn) [inline styles.css]: ## Using the Same Rule Repetitively[1]
- [x] Create a class once and the use in each HTML tag where you need it.
- [x] Delete all your `display:flex;` properties in entire CSS file
- [x] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,500 @@
...
+.flex-box-3 {
+  background-color: var(--dark-gray);
+}
+
+.flex-box-5 {
+  background-color: var(--silver);
+}
+
+.flex-box-6 {
+  background-color: var(--blue-gray);
```
Comment (bertil291utn) [inline styles.css:227]: ## Grayscale

All the elements and element's backgrounds and font colors should be grayscale color (black, white or any shade of gray), this rule also applies to images if the students decide to add any ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/grayscale_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown))

- [x] Remove all blues and green gray just keep gray with B/N colors

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,500 @@
...
+  flex-wrap: wrap;
+  width: 80%;
+  margin: 0 auto;
+  justify-content: space-evenly;
+} */
+
+/* footer ul li {
+  margin: 0 15px;
+} */
+
```
Comment (bertil291utn) [inline styles.css]: ## Inline comments

Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

### Grayscales 

- [x] Change to grayscale colors all elements you have colors in the entire page, not just these two pictures

All the elements and element's backgrounds and font colors should be grayscale color (black, white or any shade of gray), this rule also applies to images if the students decide to add any ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/grayscale_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown))

image
image

### Article tag

It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

- [x] Change all elements inside latest post and latest post-2  sections tags to `<article>` tag instead just divs

### Summary

-  Don't forget to work with inline comments below this comment box.

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: index.html (hunk 1 above)
Comment (LamiaSristy) [reply index.html]: It's done.

File: index.html (hunk 2 above)
Comment (LamiaSristy) [reply index.html]: It's done.

File: index.html (hunk 3 above)
Comment (LamiaSristy) [reply index.html]: It's done.

File: styles.css (hunk 4 above)
Comment (LamiaSristy) [reply styles.css]: It's done.

File: styles.css (hunk 5 above)
Comment (LamiaSristy) [reply styles.css:227]: It's done.

File: styles.css (hunk 6 above)
Comment (LamiaSristy) [reply styles.css]: It's done.

**Comment by bertil291utn** [comment]
**Status:** Approved ✅

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.



<h2 align="center">Ship it </h2>

<p align="center">image</p>

Please merge to master branch and move on with the next assignment.

You're doing  a great job keep going 💻 🚀 🔥

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Pull request merged**